src/components/Nav/NavBottom.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FaCaretDown } from 'react-icons/fa';
+
+function NavBottom() {
+  return (
+    <Nav>
+      <WrapList>
+        <WrapCategory>
+          <Category>전체 카테고리</Category>
+          <FaCaretDown />
+        </WrapCategory>
+        <List>구독</List>
+        <List>BEST</List>
+        <List>이벤트</List>
+        <List>101아카데미</List>
+        <List>신규 클래스</List>
+        <List>시그니처</List>
+      </WrapList>
+    </Nav>
+  );
+}
+
+const Nav = styled.div`
+  ${props => props.theme.variables.flex('row', 'center', 'center')}
+  position: fixed;
+  margin: 78px 0 0 0;
```

Comment:
```suggestion
  margin-top: 78px;
```

요소의 윗부분만 margin을 주고 싶다면 위와 같은 방식으로 하는 것이 더 직관적이겠네요!